src/server/views/help/get-support/_template.html

```diff
@@ -0,0 +1,23 @@
+<h1 class="govuk-heading-l">Get help with this form</h1>
+<h2 class="govuk-heading-s">Telephone</h2>
+<p class="govuk-body">If you have a unique reference number, have it with you when you call.</p>
+<ul class="govuk-list">
+  <li>Telephone: 020 7946 0101</li>
+  <li>Textphone: 020 7946 0102</li>
+  <li>Monday to Friday, 8am to 6pm</li>
+  <li>Saturday and Sunday, 10am to 4pm</li>
+  <li class="govuk-!-margin-top-5">Welsh language: 020 7946 0103</li>
+  <li>Monday to Friday, 8:30am to 5pm</li>
+</ul>
+<p class="govuk-body">
+  <a class="govuk-link" href="#">Find out about call charges</a>
```

Comment:
Got quite a few placeholder `href="#"` attributes here, shall we put the PR in draft?

Reply:
This is just a sample file, a developer would take this, copy it as `{slug}.html` and apply the changes requested by Naz or the content team. That's why I renamed it to `_template.html` - it won't ever be shown to a user as it won't match against a form, it's just used as a file to duplicate and then edit.

The links aren't supposed to work in this file, that's the responsibility of the developer when they create the final file for the form - they'd adjust the content as it may change between forms.

I've added a comment at the top of the file to clarify the purpose.

Reply:
I'll check about call charges in the morning as I don't think that changes between forms, I can add it if so.

Reply:
Done for call charges